src/cli/templates.hh

```diff
@@ -1542,13 +1539,10 @@ width: 1024
 # The icon to use for identifying your app on MacOS.
 # mac_icon:
 
-# TODO description & value
-# mac_sign:
-
-# TODO description & value
+# macOS codesign certificate name.
 # mac_codesign_identity:
 
-# TODO description & value
+# Paths to the files that should be included in the app bundle.
```

Comment:
This is actually the files that need to be *signed* in the bundle, not included, that's what build is for.